sdk/core/azure-core/src/main/java/com/azure/core/http/rest/QuerySubstitution.java

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.http.rest;
+
+/**
+ * A Query Substitution is a value that can be used to replace placeholder values in a URL
+ * in the query.
+ * Placeholders look like: "http://host.com/filename.html?{querParameter1}&{querParameter1}",
+ * where "{querParameter1}" and "{fileName}" are the placeholders.
```

Comment:
Even though this isn't public API, there are a few things to fix here:

1. An ampersand in JavaDoc will result in incorrect output when the HTML version is generated, seeing as & is a HTML escape code. You probably mean to use &amp;. 
2. I also note that there is a typo three times with the word 'quer' instead of 'query'. 
3. Finally, you refer to {fileName} being a placeholder but you never have a {fileName} in the sample.

Reply:
All fixed.